### DISCUSSION TITLE
Utilizing a local SSL certification in AWS Lambda

### DISCUSSION THREAD

**Discussion by mgntst** [serverless/serverless#9789] [General]
When I utilize the requests module and submit a GET request to a website the lambda function crashes. It simply says:
[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate

I can get my entire program to work when I bypass SSL verification.

This issue has been fixed. It was an issue with the requests module.

**Comment by pgrzesik** [comment]
It looks like an issue with your Lambda code, not with the Framework itself. I'm going to move this to Discussions as I feel it's better suited there than in issues
